py/desispec/preproc.py

```diff
@@ -469,6 +469,38 @@ def preproc(rawimage, header, primary_header, bias=True, dark=True, pixflat=True
 
     readnoise = np.zeros_like(image)
 
+    #- Load dark
+    dark_info=cfinder.data['DARK'].lower().replace('/','-').replace('_','-').split('-')
```

Comment:
Test first if cfinder.data has a 'DARK' entry. If not and the argument dark==None, skip dark subtraction.
Also I find fragile to rely on the formatting of the  filename to decide on the next action. 
I suggest we add instead a new keyword in the yaml file, like DARKARRAY or any other keyword that would differentiate this type of file from a traditional DARK.